### PR TITLE
Improve storagestore debug output

### PR DESCRIPTION
**Pull request by lrettig**
Print the storage key and value being stored as well as the memory addresses

### CODE REVIEW

**Comment by jakelang** [comment]
@axic can you give @lrettig write access to this repo? Working with forks is insufferable

File: src/eei.cpp [hunk 1]
```diff
@@ -562,14 +562,16 @@ inline int64_t maxCallGas(int64_t gas) {
...
-      HERA_DEBUG << "storageStore " << hex << pathOffset << " " << valueOffset << dec << "\n";
+      HERA_DEBUG << "storageStore offsets path " << hex << pathOffset << " value " << valueOffset << dec << "\n";
 
       ensureCondition(!(msg.flags & EVMC_STATIC), StaticModeViolation, "storageStore");
 
       evmc_uint256be path = loadUint256(pathOffset);
       evmc_uint256be value = loadUint256(valueOffset);
       evmc_uint256be current;
 
+      HERA_DEBUG << "storageStore contents path " << hex << toHex(path) << " value " << toHex(value) << dec << "\n";
```
Comment (jakelang) [inline src/eei.cpp]: `toHex` is only defined when built with debugging, causing build to fail in release mode.

Comment (lrettig) [reply src/eei.cpp]: What about https://github.com/lrettig/hera/blob/6a88a35fffce35f0fce389c042100ace9a97e3c3/src/eei.cpp#L194?

Comment (jakelang) [reply src/eei.cpp]: @lrettig that's in the debug module which is only compiled in debug mode

Comment (lrettig) [reply src/eei.cpp]: I find this insufferable. Lol. Either toHex should be moved out of the #ifdef, or else all `HERA_DEBUG` statements should be compiled out in prod mode. Anyway this isn't critical so I'm gonna close this for now.

**Pull request closed without merging**